### PR TITLE
Comparison Modal

### PR DESCRIPTION
**Pull request by Pakiavelli**
- Added button that toggles comparison modal
- Removed unnecessary state hooks

### CODE REVIEW

File: client/src/components/related/Card.jsx [hunk 1]
```diff
@@ -37,6 +40,11 @@ export default function Card({ product }) {
     return stylesResults[defaultStyle].original_price;
   };
 
+  const showComparison = () => {
+    setModal(!modal)
+    console.log('clicked')
```
Comment (claire123c) [inline client/src/components/related/Card.jsx:45]: don't forget to remove console.log here later

File: client/src/components/related/Card.jsx [hunk 2]
```diff
@@ -94,6 +102,8 @@ export default function Card({ product }) {
 
   return (
     <div>
+      {modal ? <Modal /> : null}
```
Comment (claire123c) [inline client/src/components/related/Card.jsx:105]: Nice use of ternary operator

File: client/src/components/related/PreviewImage.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React, { useState, useEffect } from 'react';
+
+export default function PreviewImage({ image }) {
```
Comment (claire123c) [inline client/src/components/related/PreviewImage.jsx:3]: Nice object deconstruction

File: client/src/components/related/PreviewImage.jsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  <img src={image} alt="Primary Product" />;
```
Comment (claire123c) [inline client/src/components/related/PreviewImage.jsx:4]: For the alt, do you want this to contain a description of what the image is case it doesn't show up? (like "black boots")

**Review by claire123c — COMMENTED** [review]
Looks good! Merged.

File: client/src/components/related/Card.jsx (hunk 1 above)
Comment (Pakiavelli) [reply client/src/components/related/Card.jsx:45]: image

File: client/src/components/related/PreviewImage.jsx (hunk 4 above)
Comment (Pakiavelli) [reply client/src/components/related/PreviewImage.jsx:4]: Thank you! Will change in future pull request

File: client/src/components/related/PreviewImage.jsx (hunk 3 above)
Comment (Pakiavelli) [reply client/src/components/related/PreviewImage.jsx:3]: Thank you chef.

File: client/src/components/related/Card.jsx (hunk 1 above)
Comment (Pakiavelli) [reply client/src/components/related/Card.jsx:45]: Thank you! I will remove.

**Pull request merged**